paddle/fluid/operators/math/cross_entropy.h

```diff
@@ -26,9 +27,12 @@ struct TolerableValue {
   HOSTDEVICE T operator()(const T& x) const {
```

Comment:
Is TolerableValue really a good idea?

Reply:
No, actually. We can calculate softmax and cross entropy at the same time, rather than calculate separately.
But It may require lots of changes. So this PR is just a temporary fix of wrong losses.

Reply:
@qingqing01 can make this explicitly configurable by the user. So we don't do this secretly.

Reply:
or we can calculate them at the same time